app/assets/stylesheets/modules/_nav.styl

```diff
@@ -1,6 +1,6 @@
 nav.top-nav
   top 0
-  z-index 100
+  z-index 10000
```

Comment:
Okay. Let's leave that alone for now then. The banner behavior has some more general issues that will require some CSS reworking, and this behavior will be part of that.

Also, can you make a screenshot and add an issue?